cfme/tests/automate/test_common_methods.py

```diff
@@ -189,11 +189,11 @@ def test_miq_password_decrypt(klass):
 
 @pytest.mark.tier(1)
 @pytest.mark.meta(automates=[1700524])
-@pytest.mark.ignore_stream("5.10")
 def test_service_retirement_from_automate_method(request, generic_catalog_item, custom_instance):
```

Comment:
There is a customer portal case assigned to the BZs you mentioned, can you confirm if this is a customer case, and if so, can you add `@pytest.mark.customer_scenario`?